PSL_catalog.json

```diff
@@ -0,0 +1,156 @@
+{
+    "project_one_line": {
+        "start_header": null,
+        "end_header": null,
+        "source": "https://github.com/open-source-economics/OG-USA",
+        "type": "html",
+        "data": "<p>Dynamic Tax Scoring Model</p>"
+    },
+    "key_features": {
+        "start_header": null,
+        "end_header": null,
+        "source": null,
+        "type": null,
+        "data": null
+    },
+    "project_overview": {
+        "start_header": "OG-USA",
+        "end_header": "Disclaimer",
+        "source": "README.md",
+        "type": "github_file",
+        "data": null
+    },
+    "citation": {
+        "start_header": "Citing OG-USA",
+        "end_header": null,
+        "source": "README.md",
+        "type": "github_file",
+        "data": null
+    },
+    "license": {
+        "start_header": null,
+        "end_header": null,
+        "source": "https://github.com/open-source-economics/OG-USA/blob/master/LICENSE.md",
+        "data": "<p>CC0 1.0 Universal (CC0 1.0) Public Domain Dedication",
+        "type": "html"
+    },
+    "user_documentation": {
+        "start_header": null,
+        "end_header": null,
+        "source": "https://github.com/open-source-economics/OG-USA/blob/master/documentation/OGUSAdoc.pdf",
+        "type": "html",
+        "data": "<a href=\"https://github.com/open-source-economics/OG-USA/blob/master/documentation/OGUSAdoc.pdf\">Documentation</a>"
+    },
+    "user_changelog": {
+        "start_header": null,
+        "end_header": null,
+        "source": null,
+        "type": null,
+        "data": null
+    },
+    "user_changelog_recent": {
+        "start_header": null,
+        "end_header": null,
+        "source": null,
+        "type": null,
+        "data": null
+    },
+    "dev_changelog": {
+        "start_header": null,
+        "end_header": null,
+        "source": null,
+        "type": null,
+        "data": null
+    },
+    "disclaimer": {
+        "start_header": "Disclaimer",
+        "end_header": "Using/contributing to OG-USA",
+        "source": "README.md",
+        "type": "github_file",
+        "data": null
+    },
+    "user_case_studies": {
+        "start_header": null,
+        "end_header": null,
+        "source": null,
+        "type": null,
+        "data": null
+    },
+    "project_roadmap": {
```

Comment:
What would go here?

Reply:
My understanding is that this section is for what, if any, milestones there are for the project. So maybe something like finishing the PolicyBrain API or adding some new capability. You can see the PSL roadmap as an example [here](https://github.com/open-source-economics/PSL-Core/blob/master/Core/Community/roadmap.md).